packages/mui-material/src/Rating/Rating.test.js

```diff
@@ -119,6 +120,41 @@ describe('<Rating />', () => {
     expect(container.querySelector('.customized')).to.have.tagName('label');
   });
 
+  it('should apply labelEmptyValueActive styles from theme', () => {
+    const theme = createTheme({
+      components: {
+        MuiRating: {
+          styleOverrides: {
+            labelEmptyValueActive: {
+              height: '120px',
+            },
+          },
+        },
+      },
+    });
+    const { container } = render(
+      <ThemeProvider theme={theme}>
+        <Rating value={null} />
+      </ThemeProvider>,
+    );
+
+    expect(container.querySelector(`.${classes.labelEmptyValueActive}`)).to.equal(null);
+
+    act(() => {
+      const noValueRadio = screen.getAllByRole('radio').find((radio) => {
+        return radio.checked;
+      });
+
+      noValueRadio.focus();
+    });
+    expect(container.querySelector(`.${classes.labelEmptyValueActive}`)).to.have.tagName('label');
+    expect(
+      window
+        .getComputedStyle(container.querySelector(`.${classes.labelEmptyValueActive}`))
+        .getPropertyValue('height'),
+    ).to.have.equal('120px');
+  });
```

Comment:
We should test this in browser and skip the test in JS dom.
```suggestion
    expect(container.querySelector(`.${classes.labelEmptyValueActive}`)).toHaveComputedStyle({
      height: '120px',
    });
```

Reply:
@sai6855 This comment is pending.

Reply:
Done